src/main/kotlin/uk/gov/justice/digital/assessments/restclient/AssessmentUpdateRestClient.kt

```diff
@@ -57,6 +59,7 @@ class AssessmentUpdateRestClient {
       }
   }
 
+  @Authorized(roleChecks = [Roles.OFF_ASSESSMENT_CREATE])
```

Comment:
Annotation that will trigger the aspect to secure the creation of an assessment for a given offender